### PR TITLE
[14.0][IMP] shopinvader: Use correct method to get report

### PR DESCRIPTION
**Pull request by rousseldenis**

### CODE REVIEW

**Review by sebastienbeau — APPROVED** [review]
LGTM code review

File: shopinvader/services/abstract_download.py [hunk 1]
```diff
@@ -69,7 +69,7 @@ def _get_binary_content(self, target, params=None):
         target_report_def = self._get_report_action(target, params=params)
         report_name = target_report_def.get("report_name")
         report_type = target_report_def.get("report_type")
-        report = self._get_report(report_name, report_type)
+        report = self.env["ir.actions.report"]._get_report_from_name(report_name)
```
Comment (simahawk) [inline shopinvader/services/abstract_download.py:72]: what about `_get_report` then? Remove? Deprecate? Fix?

Comment (rousseldenis) [reply shopinvader/services/abstract_download.py:72]: Mmmh, yes indeed. the good approach should have been modifying that method instead no ?

Comment (lmignon) [reply shopinvader/services/abstract_download.py:72]: @rousseldenis Should be nice to finalize this PR. IMO, you should fix `_get_report` and deprecate the usage of the `report_type` parameter (IOW, log a warning if the method is called with this report_type parameter).

**Review by lmignon — CHANGES_REQUESTED** [review]

**Comment by lmignon** [comment]
ping @rousseldenis

**Comment by sebastienbeau** [comment]
ping @rousseldenis

**Pull request closed without merging**